proxyserver/src/test/java/io/trino/gateway/proxyserver/TestProxyServer.java

```diff
@@ -79,13 +80,50 @@ public void testCustomHeader() throws Exception {
     }
   }
 
+  @Test
+  public void testLongHeader() throws Exception {
+    String mockResponseText = "CUSTOM LONG HEADER TEST";
+    String mockLongHeaderKey = "HEADER_LONG";
+    // Mockserver has max 8k for HTTP Header values so test with header value larger than default 4k
+    String mockLongHeaderValue = RandomStringUtils.random(5*1024, true, true);
```

Comment:
Nit: make the header size a parameter like `int headerLength = 1*1024*1024`

Similarly to above comment, lets use something close to the Trino default max header size.

Reply:
Is there a way to increase jetty server's default maximum header size from 8K?

Reply:
As per David's request here https://github.com/jetty/jetty.project/issues/4541, Jetty allows large headers. Do you get an error?

Reply:
yes, i must be missing sth. it works fine when i deploy in production so i guess it's issue with test code.
![image](https://github.com/trinodb/trino-gateway/assets/25147023/dc3eae89-0581-4e34-ac5a-48deec5370ef)

Reply:
Ok yeah, maybe its due to using MockWebServer, even though the error appears to be from jetty. This might be better to sort out once we start migrating away from mocks. If you can't get it working, maybe just open an issue to improve the test so we don't lose track of it.

Reply:
I created an issue here :) 
- https://github.com/trinodb/trino-gateway/issues/72